easybuild/easyconfigs/t/Trinity/Trinity-2.5.1-intel-2017a.eb

```diff
@@ -0,0 +1,41 @@
+name = 'Trinity'
+version = '2.5.1'
+
+homepage = 'http://trinityrnaseq.github.io'
+description = """Trinity represents a novel method for the efficient and robust de novo reconstruction
+ of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,
+ Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads."""
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'optarch': True}
+preinstallopts = "JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8 "
```

Comment:
This should be moved down, below the `dependencies` block (with an empty line above).

Maybe also include a comment why this is needed?

Reply:
This option did not work, I forgot to remove it. I had to set it in the easyblock file, see
https://github.com/easybuilders/easybuild-easyblocks/pull/1302